config/locales/en.yml

```diff
@@ -151,7 +151,7 @@ en:
 
       %{link} if you think this is incorrect.
     body_html_org_restricted: Your GOV.UK publishing team is managing the creation of forms for your department. Please contact them to discuss your form needs.
-    link_text: Contact the GOV⁠.⁠UK Forms team
+    link_text: Contact the GOV.UK Forms team
```

Comment:
This is interesting, this is what it looked like in my IDE. 
![image](https://github.com/alphagov/forms-admin/assets/3441519/1034bbe7-a3d9-4146-ad44-908030d4d954)

Reply:
These are word joiner characters, used to prevent line breaks between 'GOV', '.', and 'UK' [[1](https://en.wikipedia.org/wiki/Word_joiner)].

Reply:
Removed the commit now.